src/style.css

```diff
@@ -1 +1,67 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir-Book, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar-input {
+  width: 100%;
+  height: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  box-sizing: border-box;
+}
+
+.search-bar-input--top {
+  padding-left: 62px;
+  height: 70px;
+  background-position: 26px center;
+  background-size: 19px 19px;
+  font-size: 16px;
+}
+
+.search-bar-input--bottom {
+  padding-left: 33px;
+  height: 42px;
+  background-position: top 15px left 13px;
+  background-size: 11px 11px;
+  font-size: 14px;
+}
```

Comment:
It is recommended not to fix container sizes unless there's a specific requirement for it. Ensure that the fixed heights for the '.search-bar-input--top' and '.search-bar-input--bottom' are indeed necessary and that the content does not overflow or create an accidental scrollbar.